install.md

```diff
@@ -773,6 +776,155 @@ To install SCP Toolkit:
+     | Creating cluster admin role 'source-controller-tap-install-cluster-role'
+     | Creating cluster role binding 'source-controller-tap-install-cluster-rolebinding'
+     \ Creating package resource
+     | Package install status: Reconciling
+     
+      Added installed package 'source-controller' in namespace 'tap-install'
+     ```
+
+ 3. Verify the package install by running:
+
+     ```bash
+     $ tanzu package installed get source-controller -n tap-install
+      Retrieving installation details for sourcer-controller...
+      NAME:                    sourcer-controller
+      PACKAGE-NAME:            controller.source.apps.tanzu.vmware.com
+      PACKAGE-VERSION:         0.1.2
+      STATUS:                  Reconcile succeeded
+      CONDITIONS:              [{ReconcileSucceeded True  }]
+      USEFUL-ERROR-MESSAGE:
+      ```
+
+     ```bash
+     kubectl get pods -n source-system 
+     ```
+     For example:
+     ```bash
+     $ kubectl get pods -n source-system
+     NAME                                        READY   STATUS    RESTARTS   AGE
+     source-controller-manager-f68dc7bb6-4lrn6   1/1     Running   0          45h
+     ```
+     STATUS should be `Running`.
+
+## <a id='install-service-bindings'></a> Install Service Bindings 
+
+ To install Service Bindings:
+
+ 1. Follow the instructions in [Install Packages](#install-packages) above.
+
+ 2. Install the package:
+
+     ```bash
+     $ tanzu package install service-bindings -p service-bindings.labs.vmware.com -v 0.5.0 -n tap-install
+     / Installing package 'service-bindings.labs.vmware.com'
+     | Getting namespace 'tap-install'
+     - Getting package metadata for 'service-bindings.labs.vmware.com'
+     | Creating service account 'service-bindings-tap-install-sa'
+     | Creating cluster admin role 'service-bindings-tap-install-cluster-role'
+     | Creating cluster role binding 'service-bindings-tap-install-cluster-rolebinding'
+     \ Creating package resource
+     | Package install status: Reconciling
+     
+      Added installed package 'service-bindings' in namespace 'tap-install'
+     ```
+
+ 3. Verify the package install by running:
+
+     ```bash
+     $ tanzu package installed get service-bindiings -n tap-install
+     - Retrieving installation details for service-bindings...
+     NAME:                    service-bindings
+     PACKAGE-NAME:            service-bindings.labs.vmware.com
+     PACKAGE-VERSION:         0.5.0
+     STATUS:                  Reconcile succeeded
+     CONDITIONS:              [{ReconcileSucceeded True  }]
+     USEFUL-ERROR-MESSAGE:
+     ```
+
+     ```bash
+     kubectl get pods -n service-bindings 
+     ```
+     For example:
+     ```bash
+     $ kubectl get pods -n source-system
+     NAME                       READY   STATUS    RESTARTS   AGE
+     manager-6d85fffbcd-j4gvs   1/1     Running   0          22s
+     ```
+     STATUS should be `Running`.
+
```

Comment:
Just adding one comment that @heyjcollins suggested. on the next section `Verify the Installed Packages` can you add our packages on the example?  

```
      service-bindiings            service-bindings.labs.vmware.com                      0.5.0                            Reconcile succeeded
      source-controller            controller.source.apps.tanzu.vmware.com               0.1.2                            Reconcile succeeded
      convention-controller        controller.conventions.apps.tanzu.vmware.com          0.4.2                           Reconcile succeeded
```